src/Components/GameBoard.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from "react";
```

Comment:
We need comments in this file for the code that we have added. We should also add comments to the code for the Digraph that was already there indicating that it was not added by us.